melissa/actions/system_status.py

```diff
@@ -18,7 +18,7 @@ def system_status(text):
     disk_usage = psutil.disk_usage('/')
     disk_percent = disk_usage.percent
     response = "I am currently running on %s version %s. " % (os, version)
-    response += "This system is named %s and has %s CPU cores. " % (name, cores)
+    response += "This system is named %s and has %s CPU cores." % (name, cores)
     response += "Current CPU utilization is %s percent. " % cpu_percent
     response += "Current memory utilization is %s percent." % memory_percent
```

Comment:
Or if that's too much of an issue, just save the file that you made the changes in somewhere, delete your fork, re-fork and submit the changes as a single commit.